### PR TITLE
ZEN-33165 add unit test

### PR DESCRIPTION
**Pull request by romanbakaleyko**
add unit test for the fix https://github.com/zenoss/zenoss-prodbin/pull/3985/files

### CODE REVIEW

File: Products/ZenModel/tests/testIpNetwork.py [hunk 1]
```diff
@@ -254,6 +255,20 @@ def assertZPropertiesExist( network ):
...
+    def testFindStaleNetOrIp(self):
+        """
+        If we have a record in cache which is not in database it 
+        should be removed from cache and return None when attempt to get it 
+        """
+        dmdNet = self.dmd.Networks
+        # Populate cache without adding object to database
+        dmdNet.get_network_cache().add_net(IpNetwork('1.1.1.0', '24'))
+        net = dmdNet.findNet('1.1.1.0', '24')
+        self.assert_(net == None)
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/tests/testIpNetwork.py:267]: The "pythonic" way to write this test is

```
self.assertIsNone(net)
```
If, for some reason, the `assertIsNone` method is unavailable, then use the `is` operator, e.g.
```
self.assert_(net is None)
```

File: Products/ZenModel/tests/testIpNetwork.py [hunk 2]
```diff
@@ -254,6 +255,20 @@ def assertZPropertiesExist( network ):
...
+    
+        dmdNet.get_network_cache().add_net(IpNetwork('2.2.2.0', '24'))
+        ip = dmdNet.findIp('2.2.2.2', '24')
+        self.assert_(ip == None)
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/tests/testIpNetwork.py:271]: Same here.

**Review by jpeacock-zenoss — CHANGES_REQUESTED** [review]

File: Products/ZenModel/tests/testIpNetwork.py (hunk 1 above)
Comment (romanbakaleyko) [reply Products/ZenModel/tests/testIpNetwork.py]: done

File: Products/ZenModel/tests/testIpNetwork.py (hunk 2 above)
Comment (romanbakaleyko) [reply Products/ZenModel/tests/testIpNetwork.py]: done

**Review by jpeacock-zenoss — APPROVED** [review]

**Pull request merged**